k8s/upgrade/src/bin/upgrade-job/helm/chart.rs

```diff
@@ -1024,6 +1035,12 @@ enum PromtailInitContainer {
     InitContainer(Vec<Container>),
 }
 
+impl Default for PromtailInitContainer {
+    fn default() -> Self {
+        Self::InitContainer(Vec::<Container>::new())
+    }
+}
```

Comment:
Use derive Default
```suggestion
```

Reply:
Couldn't derive:
```
error: the `#[default]` attribute may only be used on unit enum variants
    --> k8s/upgrade/src/bin/upgrade-job/helm/chart.rs:1023:5
     |
1023 |     InitContainer(Vec<Container>),
     |     ^^^^^^^^^^^^^
     |
     = help: consider a manual implementation of `Default`
```
Using the manual default implementation.